audit_trail_integration_test.go

```diff
@@ -0,0 +1,94 @@
+//go:build integration
+// +build integration
+
+package tfe
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAuditTrailRead(t *testing.T) {
+	skipIfFreeOnly(t)
+	skipIfNoOrgToken(t)
+
+	client := testClient(t)
+	ctx := context.Background()
+
+	orgToken := os.Getenv("TFE_ORGANIZATION_TOKEN")
```

Comment:
Why did you decide that creating an org token would not work? I'm sorry I forgot.

Reply:
Because we currently have no way in go-tfe to upgrade an organization to the business tier. The integration tests in atlas do in fact upgrade the entitlement set for an org but it's an internal endpoint as I understand it. 

I discussed with @taiidani and @annawinkler on potentially configuring tflocal to default any organization that's created to the business plan so we can run the entire test suite. This is non trivial and requires further internal discussion. 👍

Reply:
The non go-tfe endpoint that the Atlas integration tests use: https://github.com/hashicorp/atlas/blob/32947de5536a9df4fb79ce18868b0b4ad29bfa07/integration-tests-api/internal/testenv/helpers.go#L775-L838